### PR TITLE
Raise timeout for ES cleanup script

### PR DESCRIPTION
**Pull request by mh-park**

### CODE REVIEW

File: plugin/storage/es/esCleaner.py [hunk 1]
```diff
@@ -18,14 +19,15 @@ def main():
 
     for index in ilo.working_list():
         print "Removing", index
-    delete_indices = curator.DeleteIndices(ilo)
```
Comment (black-adder) [inline plugin/storage/es/esCleaner.py]: can we make this configurable and have it default to 120?

File: plugin/storage/es/esCleaner.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 import curator
+import logging
 import sys
+import os
 
 
 def main():
     if len(sys.argv) == 1:
-        print('USAGE: %s NUM_OF_DAYS HOSTNAME[:PORT] ...' % sys.argv[0])
+        print('USAGE: [TIMEOUT=(default 120)] %s NUM_OF_DAYS HOSTNAME[:PORT] ...' % sys.argv[0])
```
Comment (yurishkuro) [inline plugin/storage/es/esCleaner.py]: print a brief explanation of the script and explain all parameters

Comment (mh-park) [inline plugin/storage/es/esCleaner.py]: this happens in `./es_indices_clean.sh`
technically i don't need this piece of code in the python script, but should i add an explanation to both anyways?

Comment (mh-park) [inline plugin/storage/es/esCleaner.py]: https://github.com/uber/jaeger/blob/b31ca51b47d7ca7c9e78fbe6fd64638fedf30175/plugin/storage/es/es_indices_clean.sh#L4

File: plugin/storage/es/esCleaner.py [hunk 3]
```diff
@@ -18,14 +21,16 @@ def main():
...
+    timeout = int(os.getenv("TIMEOUT", 120))
+    delete_indices = curator.DeleteIndices(ilo, master_timeout=timeout)
     delete_indices.do_action()
 
 
 def empty_list(ilo, error_msg):
     try:
         ilo.empty_list_check()
     except curator.NoIndices:
+        logging.info(error_msg)
```
Comment (black-adder) [inline plugin/storage/es/esCleaner.py]: why is this needed if print is happening below?
